charts/jenkins/unittests/config-test.yaml

```diff
@@ -15,6 +15,7 @@ tests:
             app.kubernetes.io/instance: RELEASE-NAME
             app.kubernetes.io/managed-by: Helm
             app.kubernetes.io/name: jenkins
+            helm.sh/chart: jenkins-5.8.73
```

Comment:
this change here is breaking all renovate PRs to keep dependencies up to date.

I've had a bit of a look but haven't found a fix yet.

@cpcwood are you able to check this please?

I'll revert in a few days if you aren't able to.

Reply:
Sorry missed this, I'll take a look tomorrow

Reply:
@timja https://github.com/jenkinsci/helm-charts/pull/1444